### PR TITLE
Add Barcelona file to class repo

### LINKED ISSUE #27: castuo tracks progress on GitHub Flow
- [x] create a branch (git)
- [x] add some commits (git)
- [x] open a pull request (gitHub)
- [x] additional commits (git)
- [ ] merge (git)

### PR DESCRIPTION
**Pull request by castuo**
This PR adds a Barcelona file to the class repo. This PR closes #27 

- @ruthtubau do you know any local good attractions in Barcelona?
- What's your favorite place in Barcelona?

### CODE REVIEW

File: castuo-barcelona.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Barcelona, ES
+
+### Restaurants
+
```
Comment (ruthtubau) [inline castuo-barcelona.md]: what's your favorite restaurant?

Comment (castuo) [inline castuo-barcelona.md]: PorSant of course!!!

**Comment by sebastianvendo** [comment]
Ginza!

File: castuo-barcelona.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+### Attractions for Tourist
+
+- Parc d'atraccions de Montjuic
```
Comment (darrenob) [inline castuo-barcelona.md]: This park is not PCI compliant

Comment (castuo) [inline castuo-barcelona.md]: @darrenob why not? Did you have any issue

**Comment by CarlosVendo** [comment]
Porsant! que raro 😄 	ccf257b

**Comment by darrenob** [comment]
:shipit:

**Pull request merged**